documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc

```diff
@@ -1049,51 +1049,59 @@ include::{extrasdir}/collections-map-bidirectional-example.sql[]
 When discussing arrays, it is important to understand the distinction between SQL array types and Java arrays that are mapped as part of the application's domain model.
 
 Not all databases implement the SQL-99 ARRAY type and, for this reason,
-Hibernate doesn't support native database array types.
+the SQL type used by Hibernate for arrays varies depending on the database support.
 
-Hibernate does support the mapping of arrays in the Java domain model - conceptually the same as mapping a List.
-However, it is important to realize that it is impossible for Hibernate to offer lazy-loading for arrays of entities and, for this reason,
-it is strongly recommended to map a "collection" of entities using a List rather than an array.
+NOTE: It is impossible for Hibernate to offer lazy-loading for arrays of entities and, for this reason,
+it is strongly recommended to map a "collection" of entities using a List or Set rather than an array.
 
-[[collections-array-binary]]
-==== Arrays as binary
+[[collections-array-as-basic]]
+==== Arrays as basic value type
```

Comment:
Please don't break anchors :( People linking to `/current/` or `/stable/` would get what essentially amounts to a dead link.

If you really want to change the anchor, something like this should work (assuming you're using the latest version of Asciidoctor):

```suggestion
[[collections-array-as-basic]]
==== [[collections-array-binary]] Arrays as basic value type
```